python/tk_multi_workfiles/entity_models/deferred_model.py

```diff
@@ -519,6 +520,23 @@ def _on_deferred_data_refreshed(self, sg_entity, changed, pending_refresh=False)
                 # The item might be already gone because one its ancestor was
                 # deleted.
                 self._delete_item(item)
+        # If we have any uid which is not in both sets, then things changed.
+        if existing_uids ^ refreshed_uids:
```

Comment:
For clarity, i suggest using the verbose form here: [`s.symmetric_difference(t)`](https://docs.python.org/2/library/sets.html). I suspect most readers of the code are not familiar with what the `^` operator is doing for a set.